source/Octopus.Server.Client/Model/Permission.cs

```diff
@@ -275,6 +275,8 @@ public class Permission : IEquatable<Permission>
         [Description("View deployment target tags")] public static readonly Permission TargetTagView = new("TargetTagView");
         
         [Description("Create, edit, delete deployment target tags")] public static readonly Permission TargetTagAdminister = new("TargetTagAdminister");
+
+        [Description("Create deployments and runbooks that are prioritized")] public static readonly Permission PrioritizeTask = new("PrioritizeTask");
```

Comment:
nit: Might be being pedantic here... but should they be called `TaskPrioritize`, like TaskCreate/TaskCancel etc for consistency?

Reply:
Good ideas, that is more consistent